tools/meteor-services/deploy.js

```diff
@@ -71,8 +71,8 @@ const CAPABILITIES = ['showDeployMessages', 'canTransferAuthorization'];
 //   body, or a generic 'try again later' message, as appropriate
 
 function deployRpc(options) {
-  options = _.clone(options);
-  options.headers = _.clone(options.headers || {});
+  options = clone(options);
```

Comment:
In the PR you ask whether or not deep cloning is necessary, but _.clone doesn't actually deep clone (which is why there's two clones in a row here). So it should be fine to use a non-underscore shallow clone here (`Object.assign({}, x)` right?)